docker/k8s/mysqlctl/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 The Vitess Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG VT_BASE_VER=latest
+
+FROM vitess/k8s:${VT_BASE_VER} AS k8s
+
+FROM debian:buster-slim
+
+# Set up Vitess environment (just enough to run pre-built Go binaries)
+ENV VTROOT /vt
+ENV VTDATAROOT /vtdataroot
+
+# Prepare directory structure.
+RUN mkdir -p /vt/bin && \
+   mkdir -p /vt/config && mkdir -p /vtdataroot
+
+# Copy binaries
+COPY --from=k8s /vt/bin/mysqlctl /vt/bin/
+
+# Copy certs to allow https calls
+COPY --from=k8s /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+
+# copy vitess config
+COPY --from=k8s /vt/config /vt/config
```

Comment:
I'm not sure if the directory `/vt/config` is needed here?

Reply:
`vt/config` contains mysql `.cnf` files so it is probably required.